docs/spec/spec.md

```diff
@@ -2758,9 +2762,209 @@ public function main() returns error? {
 }
 ```
 
-## 13. Tools
+## 13. Federated Subgraph
 
-### 13.1 GraphiQL client
+The Ballerina GraphQL Modules provide the capability to expose a `graphql:Service` as a federation2 subgraph.
```

Comment:
Let's link the federation v2 spec link here.

Reply:
updated